### ISSUE TITLE
Bug: Focus issues for keyboard users.

### ISSUE THREAD

**Issue by terichadbourne** [ProtoSchool/protoschool.github.io#160]
I just did a quick accessibility test using only my keyboard to navigate the ProtoSchool site. 

On most pages, I'm able to tab through all nav items and links with clearly visible focus rings, except for the "Next/Submit" button at the bottom of the page which has a very light gray focus ring against a white background, with not nearly enough context. It looks like I'll be able to update the color in an existing custom class to fix that.
 
![image](https://user-images.githubusercontent.com/19171465/52741369-02c57c80-2f8a-11e9-8fb7-97b8846d9d35.png)

Within one of our standard tutorial pages, the tabbing works fine until I tab into the code editor (Monaco, from VS Code) where the user is supposed to complete code challenges. Once I'm in there, it interprets my tabbing as me wanting to add tabs to my code, and it interprets the enter key as me wanting to start a new line, so I appear to trapped inside the code editor and unable to get past it to the "submit" button that would let me validate my solution. I found an [accessibility guide](https://github.com/Microsoft/monaco-editor/wiki/Monaco-Editor-Accessibility-Guide) for Monaco editor which actually gives the solution to this problem (a special key combo to get you out of "tab trapping")
